app/src/main/java/br/org/cesar/knot_setup_app/fragment/login/LoginContract.java

```diff
@@ -0,0 +1,15 @@
+package br.org.cesar.knot_setup_app.fragment.login;
+
+public interface LoginContract {
+
+    interface ViewModel {
+        void fillEmail(String email);
+        void callbakcOninvalidCredentials();
+        void callbackOnLogin();
+    }
+
+    interface Presenter {
+        void onResume();
```

Comment:
IMO the presenter should not have a method name onResume. onResume is an Android Lifecycle specific method and the idea of the presenter is to decouple Android specific code from the rest of the code.
Just name the method after what it does (or have multiple methods if necessary).

Reply:
This discussion is happening in another comment.